src/devices/AD5328/AD5328.cs

```diff
@@ -0,0 +1,72 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+using System;
+using System.Device.Spi;
+using UnitsNet;
+
+namespace Iot.Device.DAC
+{
+    /// <summary>
+    /// Driver for the AD5328 DAC.
+    /// </summary>
+    public class AD5328 : IDisposable
+    {
+        private SpiDevice _spiDevice;
+        private ElectricPotential _referenceVoltageA;
+        private ElectricPotential _referenceVoltageB;
+        private bool _disposedValue;

```

Comment:
Even if false is the default, always good to have it explicit: ` private bool _disposedValue = false;`

Reply:
done!